selinux/qubes-gui-agent.te

```diff
@@ -0,0 +1,14 @@
+policy_module(qubes-gui-agent, 0.0.1)
+
+require {
+	type xdm_t, etc_t, pulseaudio_t;
+	class passwd rootok;
+}
+
+allow xdm_t etc_t:file { create write };
+allow xdm_t self:passwd rootok;
+pulseaudio_domtrans(xdm_t)
```

Comment:
Out of curiosity - which part requires this? Is it because of staring pulseaudio (effectively) from `/usr/bin/qubes-session`? Will moving it to systemd user unit improve the situation here?

Reply:
Yes and yes, though from a security perspective I believe it is a wash if one can use `dev_rw_xen` to get root anyway.